### ISSUE TITLE
WARNING:tensorflow:`evaluate()` received a value for `sample_weight`, but `weighted_metrics` were not provided

### ISSUE THREAD

**Issue by das22** [nadavbra/protein_bert#28]
Hi, 

I'm getting the following warning when when running the "Run all benchmarks" section of the demo notebook with some of my own data. Is this something to worry about? The fine-tuning finishes and yields results for training/test/validation. Could it be related to using CPUs not GPUs?

```WARNING:tensorflow:`evaluate()` received a value for `sample_weight`, but `weighted_metrics` were not provided.  Did you mean to pass metrics to `weighted_metrics` in `compile() ?  If this is intentional you can pass `weighted_metrics=[]` to `compile()` in order to silence this warning. ```

Also, while at it, would you recommend to experiment with any of the settings in that block (lr, dropout, batch size, etc)? I wasn't sure if any parameters are being varied since a validation set is present or if changing something manually might be of benefit.

Thank you!

**Comment by nadavbra** [comment]
@das22 I'm actually not sure. Looks like a recent change in keras. Since it's only a warning I tend to think it's probably fine. What do you run exactly when getting this warning?

**Comment by nadavbra** [comment]
About your other question, we chose settings that seemed to work reasonably well across all benchmarks, so I expect it to produce reasonable results for your data too. But of course, if you have the time, you could gain from tweaking them and further optimizing them to your data (just be careful of overfitting).

**Comment by das22** [comment]
Thanks a lot!
